service/src/test/java/de/adorsys/psd2/sandbox/xs2a/pis/PIS.feature

```diff
@@ -24,9 +30,37 @@ Feature: Payment Initiation Service
       | payment-product       | psu-id | password | sca-method | tan   | sca-status | code |
       | sepa-credit-transfers | PSU-1  | 12345    | SMS_OTP    | 54321 | finalised  | 200  |
 
+
+    ################################################################################################
+    #                                                                                              #
+    # Periodic Payment                                                                             #
+    #                                                                                              #
+    ################################################################################################
+  Scenario Outline: Initiation of a periodic  payment
+    Given PSU initiated a periodic payment using the payment product <payment-product>
+    When PSU requests the payment data
+    Then the payment data and response code <code> are received and its transaction-status is <status>
+    Examples:
+      | payment-product       | code | status |
+      | sepa-credit-transfers | 200  | RCVD   |
+
     ################################################################################################
     #                                                                                              #
-    # Payment Status                                                                               #
+    # Future-dated Payment                                                                         #
```

Comment:
Same comment as before. Reuse `Payment Initiation`

Reply:
Furthermore, we can maybe refactor the code to pass the appropriate payment-class in the feature file by passing enums like single-payment/periodic-payment/future-dated-payment. In the step-file we would then decide which object to create

Reply:
done